website/assets/scss/_home.scss

```diff
@@ -80,6 +80,18 @@
 				background-color: rgba(#62a1ff, 0.6);
 			}
 		}
+		.cta-mentorship {
```

Comment:
Kind of detail, but...

these `cta` boxes ([line 55](https://github.com/cncf/tag-contributor-strategy/pull/508/files#diff-0b447b380abaeab9c8b4e63e8f8e82788e0dff36386f7cb0ca61416d60680c7dR55)) should not have `justify-content: center;` IMO but instead should have a left padding.

Reply:
Makes sense. I've made an update to fix this. 

More fine-tuning is needed but I will wait first to make sure people are agreed on the content.